atlas/atlas.star

```diff
@@ -0,0 +1,175 @@
+"""
+A URL is an absolute location within the atlas.
+"""
+scalar URL
+
+"""
+Names are dotted paths of GraphQL identifiers, like `spotify.User`
+and `String`.
+"""
+scalar Name
+
+type Query {
+  at(id: URL): Point
+  edge(name: Name, from: Name, to: Name): Edge
+  type(id: URL): Type
+  interface(id: URL): Interface
+  edge(id: URL): Edge    
+  points: [Point]
```

Comment:
We should probably think about pagination early on in this design as it always bites us trying to add it later. Especially as we deal with massive graphs